src/main/java/fr/lunatech/timekeeper/services/UserService.java

```diff
@@ -42,7 +42,7 @@
     OrganizationService organizationService;
 
     public Optional<UserResponse> findResponseById(Long id, AuthenticationContext ctx) {
-        return findById(id, ctx).map(UserResponse::bind);
+        return findUserById(id, ctx).map(UserResponse::bind);
```

Comment:
Please, don't change the name of this method for the same reasons as the `EventTemplateService`.

Reply:
may be this method should be kept private , no ?